### ISSUE TITLE
create a kubernetes specific CNCI

### ISSUE THREAD

**Issue by kaccardi** [01org/ciao#725]
create a CNCI optimized for kubernetes with load balancing and automatic routing between pod subnets.

Depends on #708

**Comment by kaccardi** [comment]
each image should contain a script to parse json config for k8s from cloud init

**Comment by kaccardi** [comment]
we changed our k8s strategy, so this is no longer relevant.
